src/models/FoodItem.js

```diff
@@ -2,15 +2,23 @@ import {Document} from "./Document"
 
 export class FoodItem extends Document{
 
-    constructor(name, photoURI){
+    /**
```

Comment:
Lmk if you are still wanting to keep this. I'm on board with not making our own collection just for food items